### ISSUE TITLE
rio up with github repo is not building resources

### ISSUE THREAD

**Issue by rancher-max** [rancher/rio#637]
**Describe the bug**
Running `rio up <gh repo url>` is not registering the Riofile in the repo and therefore not building any of the resources defined in it.

**To Reproduce**
1. Run `rio up https://github.com/rancher-max/riofile-demo`

**Expected behavior**
It should check the repo for a Riofile and build the services, routers, etc. that are defined in it.

**Kubernetes version & type (GKE, on-prem)**: `kubectl version` Using GKE cluster since that is what works with GH builds on rio v0.6.0-alpha.1
```
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.2", GitCommit:"c97fe5036ef3df2967d086711e6c0c405941e14b", GitTreeState:"clean", BuildDate:"2019-10-15T19:18:23Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.7-gke.14", GitCommit:"d7951e52ecdf56a1de9e031c553569ec13849a80", GitTreeState:"clean", BuildDate:"2019-10-11T23:17:57Z", GoVersion:"go1.12.9b4", Compiler:"gc", Platform:"linux/amd64"}
```
Type:
**Rio version**: `rio info`
```
Rio Version: v0.6.0-alpha.1 (7678d67d)
Rio CLI Version: v0.6.0-alpha.1 (7678d67d)
```

**Comment by StrongMonkey** [comment]
Available to test in v0.6.0-alpha2.

**Comment by rancher-max** [comment]
Working in v0.6.0-alpha.2
